Arguments: 
  /Users/jh/.volta/tools/image/node/18.13.0/bin/node /Users/jh/.volta/tools/image/yarn/1.22.19/bin/yarn.js add ethers-v5

PATH: 
  /Users/jh/.volta/tools/image/yarn/1.22.19/bin:/Users/jh/.volta/tools/image/node/18.13.0/bin:/Users/jh/.pyenv/shims:/Users/jh/.volta/bin:/Users/jh/bin:/usr/local/bin:/Users/jh/.volta/bin:/Users/jh/bin:/usr/local/bin:/Users/jh/personal/misc/pair/bin:/Users/jh/.local/bin:/Users/jh/misc/solana/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Users/jh/.cargo/bin:/usr/local/opt:/Users/jh/.foundry/bin:/usr/local/opt:/Users/jh/.foundry/bin

Yarn version: 
  1.22.19

Node version: 
  18.13.0

Platform: 
  darwin arm64

Trace: 
  Error: https://registry.yarnpkg.com/ethers-v5: Not found
      at params.callback [as _callback] (/Users/jh/.volta/tools/image/yarn/1.22.19/lib/cli.js:66145:18)
      at self.callback (/Users/jh/.volta/tools/image/yarn/1.22.19/lib/cli.js:140890:22)
      at Request.emit (node:events:513:28)
      at Request.<anonymous> (/Users/jh/.volta/tools/image/yarn/1.22.19/lib/cli.js:141862:10)
      at Request.emit (node:events:513:28)
      at IncomingMessage.<anonymous> (/Users/jh/.volta/tools/image/yarn/1.22.19/lib/cli.js:141784:12)
      at Object.onceWrapper (node:events:627:28)
      at IncomingMessage.emit (node:events:525:35)
      at endReadableNT (node:internal/streams/readable:1359:12)
      at process.processTicksAndRejections (node:internal/process/task_queues:82:21)

npm manifest: 
  No manifest

yarn manifest: 
  No manifest

Lockfile: 
  # THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
  # yarn lockfile v1
